src/Tickets/Seating/template-tags.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Template tags defined by the Seating Tickets feature.
+ *
+ * @since TBD
+ */
+
+use TEC\Tickets\Seating\Meta;
+
+/**
+ * Returns whether the event is using assigned seating.
+ *
+ * @since TBD
+ *
+ * @param int $post_id The post ID of the event.
+ *
+ * @return bool Whether the event is using assigned seating.
+ */
+function tec_tickets_is_assigned_seating_enabled( int $post_id ): bool {
```

Comment:
Add a  check on `function_exists` here.

Reply:
I _think_ this could just be called `tec_tickets_seating_enabled`, a bit less verbose.